projects/core/components/tooltip/tooltip.component.ts

```diff
@@ -34,6 +34,11 @@ export class TuiTooltipComponent<C = any> extends TuiHintOptionsDirective {
         return this.appearance || '';
     }
 
+    @HostBinding('class._small')
+    protected get small(): boolean {
```

Comment:
Let's drop that, we should not depend on icon names, leave it always small for now.

Reply:
droped